_guides/attributes.adoc

```diff
@@ -6,6 +6,7 @@
 :graalvm-version: 19.0.2
 :surefire-version: 2.22.0
 :restassured-version: 3.3.0
+:jdk-version: JDK 8 to 11
```

Comment:
I would go with `8 or 11`. We don't really support non-LTS versions. E.g. if someone reports an issue with JDK 9, we will ask him to upgrade.